### PR TITLE
Added some markup for 404 page

### PR DESCRIPTION
**Pull request by marcustisater**
Can someone look through quickly and verify?

Nothing exciting happening :stuck_out_tongue_winking_eye: - I just want to be strict about the LGTM thing.

### CODE REVIEW

File: web_modules/layouts/PageError/index.js [hunk 1]
```diff
@@ -17,28 +19,36 @@ export default class PageError extends Component {
     const {
       error,
       errorText,
-    } = this.props 
+    } = this.props
 
     return (
-      <div className={ styles.root }>
+      <main className={ styles.root }>
```
Comment (thangngoc89) [inline web_modules/layouts/PageError/index.js]: there is no .root class

**Comment by marcustisater** [comment]
Can you try verify this by typing `lgtm` @thangngoc89? I added you as a validator now.

**Comment by thangngoc89** [comment]
lgtm

File: web_modules/layouts/PageError/index.js (hunk 1 above)
Comment (marcustisater) [inline web_modules/layouts/PageError/index.js]: Good catch, thank you

File: web_modules/layouts/PageError/index.js [hunk 2]
```diff
@@ -17,28 +19,36 @@ export default class PageError extends Component {
...
-            error === 404 &&
-              <div>
-                <p>
-                  { 'The page you requested was not found.' }
-                </p>
-              </div>
-          }
-      </div>
+
+        <div className={ styles.hero }>
```
Comment (thangngoc89) [inline web_modules/layouts/PageError/index.js]: .hero class too

**Comment by marcustisater** [comment]
thanks man, I'm out of my game today haha.

Everything looks good now?

**Comment by thangngoc89** [comment]
yup. It's enough for basic styling

**Comment by marcustisater** [comment]
Yeah, follow up here #156

**Pull request merged**